conformity/resource_conformity_profile_test.go

```diff
@@ -2,16 +2,17 @@ package conformity
 
 import (
 	"fmt"
-	"github.com/trendmicro/terraform-provider-conformity/pkg/cloudconformity"
 	"testing"
 
+	"github.com/trendmicro/terraform-provider-conformity/pkg/cloudconformity"
+
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
 func TestAccResourceconformityProfile(t *testing.T) {
 	ttl := "72"
-	ttlUpdate := "71"
+	// ttlUpdate := "71"
```

Comment:
❓ Should we put this back?
```suggestion
	ttlUpdate := "71"
```